@stellar/lyra-api/README.md

```diff
@@ -0,0 +1,16 @@
+# Lyra API
+
+## Installation
+
+Before this can be imported in another project in this monorepo, run
+`yarn build` in this directory to create the built package.
+
+## `@lyraAlias` import
+
+Because this package needs to be packaged and published, it cannot have any
+dependencides that are not on the npm registry. Therefore, instead of using Yarn
+to import the `@lyra` workspace via symlinked dependency like in other repos,
+all references to `@lyra` must be relative paths.
+
+tsconfig.json rewrites `@lyraAlias` to a relative path to `@lyra` folders to
+prevent devs from manually having to write out a relative path.
```

Comment:
so this is kind of a blindspot with Yarn workspaces, unfortunately. They really do expect you to publish everything, which I don't want to do.

Reply:
I'd like to eventually use the symlinked dep during development, and at "prepack", swap out the symlinked dep for the relative path.

A couple of npm packages exist that sorta do this, but not to a level to my liking

Reply:
also open to renaming `@lyraAlias` to something else. Just didn't want to call it `@lyra` as it would lead someone to think it was a symlinked dep like in other repos

Reply:
Cool! I'll test it out tomorrow, that sounds like it hits all the issues I noticed though. Alias seems like a good solution.